extensions/parquet/base/src/main/java/io/deephaven/parquet/base/materializers/InstantNanosFromInt96Materializer.java

```diff
@@ -32,6 +32,20 @@ public PageMaterializer makeMaterializerNonNull(ValuesReader dataReader, int num
         }
     };
 
+    private static final int BYTES_PER_VALUE = Long.BYTES + Integer.BYTES;
+
+    public static long convertValue(final byte[] data) {
+        if (data.length != BYTES_PER_VALUE) {
+            throw new IllegalArgumentException("Invalid Int96 data length: " + data.length + ", expected: "
+                    + BYTES_PER_VALUE);
+        }
+        final ByteBuffer resultBuffer = ByteBuffer.wrap(data);
+        resultBuffer.order(java.nio.ByteOrder.LITTLE_ENDIAN);
+        final long nanos = resultBuffer.getLong();
+        final int julianDate = resultBuffer.getInt();
+        return (julianDate - JULIAN_OFFSET_TO_UNIX_EPOCH_DAYS) * (NANOS_PER_DAY) + nanos + offset;
+    }
```

Comment:
Should these be tested explicitly? It looks like they are tested using canned generated test files, but a simple companion test using constant `byte[]` might be a good backup.